test_data/pod_without_resources.json

```diff
@@ -0,0 +1,67 @@
+{
+  "uid": "1299d386-525b-4032-98ae-1949f69f9cfc",
+  "kind": {
+    "group": "",
+    "kind": "Pod",
+    "version": "v1"
+  },
+  "resource": {
+    "group": "",
+    "version": "v1",
+    "resource": "pods"
+  },
+  "requestKind": {
+    "group": "",
+    "version": "v1",
+    "kind": "Pod"
+  },
+  "requestResource": {
+    "group": "",
+    "version": "v1",
+    "resource": "pods"
+  },
+  "name": "nginx",
+  "namespace": "default",
+  "operation": "CREATE",
+  "userInfo": {
+    "username": "kubernetes-admin",
+    "groups": [
+      "system:masters",
+      "system:authenticated"
+    ]
+  },
+  "object": {
+    "apiVersion": "v1",
+    "kind": "Pod",
+    "metadata": {
+      "name": "test-pod",
+      "namespace": "default",
+      "labels": {
+        "cc-center": "123",
+        "owner": "team-alpha"
+      }
+    },
+    "spec": {
+      "containers": [
+        {
+          "name": "pause",
+          "image": "registry.k8s.io/pause"
+        },
+        {
+          "name": "mycontainer",
+          "image": "image:latest",
+          "resources": {
+            "limits": {
+              "cpu": "2m",
+              "memory": "2Gi"
+            },
+            "requests": {
+              "cpu": "2m",
+              "memory": "2Gi"
+            }
```

Comment:
The file name does not match what is in the file. Should not the `resource` field be empty?

Reply:
Yes it should, I just made [this change](https://github.com/kubewarden/container-resources-policy/pull/18/commits/8c968566a702748292f061e945df3e86bfc053c1). I'm not able to run the bats tests since I can't build the policy [at the moment](https://github.com/kubewarden/container-resources-policy/issues/17#issuecomment-1966981012).

Reply:
This is resolved here https://github.com/kubewarden/container-resources-policy/pull/18/commits/8c968566a702748292f061e945df3e86bfc053c1.